applications/climatology/climatology.ipynb

```diff
@@ -0,0 +1,3533 @@
+{
```

Comment:
<p>This could be just me missing something, but is there a way we could distinguish this cell from the one in the previous section? Like, could we pull the <code>tos_anom.mean(dim=["lat", "lon"])</code> part out of the plot call to make it clear that it's a part of the calculation? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NCAR/geocat-applications/pull/71/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>